src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/AdapterUtil.cs

```diff
@@ -1330,12 +1330,6 @@ internal static Task<T> CreatedTaskWithCancellation<T>()
             return completion.Task;
         }
 
-        internal static void TraceExceptionForCapture(Exception e)
```

Comment:
What prompted removal of this one?

Reply:
Purely neatness - it's no longer referenced in either project.

Reply:
That's the best reason 👍